packages/editor/src/client/panel/peerProfileEditPanel.tsx

```diff
@@ -144,15 +204,22 @@ export function PeerInfoEditPanel({onClose, onSave}: ImageEditPanelProps) {
           onClose={() => setChooseModalOpen(false)}
           onSelect={value => {
             setChooseModalOpen(false)
-            setLogoImage(value)
+            isLogoChange ? setLogoImage(value) : setCallToActionImageID(value)
           }}
         />
       </Drawer>
 
       <Drawer show={isEditModalOpen} size={'sm'} onHide={() => setEditModalOpen(false)}>
-        {logoImage && (
-          <ImagedEditPanel id={logoImage!.id} onClose={() => setEditModalOpen(false)} />
-        )}
+        {isLogoChange
```

Comment:
this is a lot of repetitive code i think you can put must of the logic in side the `id={...}` expression.